.devcontainer/Dockerfile

```diff
@@ -45,24 +48,23 @@ RUN apt-get update \
 #     && add-apt-repository ppa:ubuntu-toolchain-r/test \
 #    && apt-get update
 
-# Install common compilation tools
-RUN apt-get -y install git build-essential pkg-config autoconf automake libtool bison flex libpq-dev parallel libunwind-dev
-
-# Update compiler tools
-RUN apt-get -y install libstdc++-7-dev clang-format-8 ccache
-
-# gcc
-RUN apt-get -y install cpp-7 gcc-7 g++-7
-# clang
-RUN apt-get -y install clang-8 llvm-8
+# Install development tools
```

Comment:
this change is actually not a good change:
running multiple `apt-get install` has no real performance benefits, on the other hand, crafting things like this forces a full image rebuild (when changing compilers) as there are no layers and as a consequence slows down a lot future updates.

Now that you figured out how to pass environment variables, I am actually tempted to switch to user selectable compiler toolchains in the devcontainer.json file (and having proper layers becomes more important)

Reply:
If the goal is to be able to run the separate apt-get installs in isolation later on, then this Dockerfile requires some modifications, because every isolated apt install requires an apt update otherwise rerunning only one layer later can error. I'll undo this part of the change and will open another PR fixing the missing apt updates.